packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceRegistrationState.ts

```diff
@@ -314,5 +318,27 @@ export class ServiceRegistrationState {
         'Service version can not be empty in Semi-interactive and Prod service type',
       );
     }
+
+    // validate service parameter multiplicity. Currently [1..*] is not allowed for service registration.
+    const queryLambda = (this.service.execution as PureExecution).func;
+    const parameters = (queryLambda.parameters ?? []) as object[];
+    if (parameters.length) {
+      const params = buildLambdaVariableExpressions(
+        queryLambda,
+        this.editorStore.graphManagerState,
+      )
+        .filter(filterByType(VariableExpression))
+        .filter(
```

Comment:
Could we make it a bit cleaner. 
Can we add a list of accepted multiplicity in the form of `TYPICAL_MULTIPLICITY_TYPE`. 
Then just validate each multiplicity is one of those types? 
This would make a bit easier to read on whats accepted and what is not.